releases.md

```diff
@@ -0,0 +1,100 @@
+# Tekton Pipeline Releases
+
+## Release Frequency
+
+Tekton Pipelines follows the Tekton community [release policy][release-policy]
+as follows:
+
+- Versions are numbered according to semantic versioning: `vX.Y.Z`
+- A new release is produced on a monthly basis
+- Four releases a year are chosen for long term support (LTS). All remaining
```

Comment:
Can we describe what LTS entails ? How many month are those supported, what type of support is there ? (security fix, bugfix, …)

Reply:
That's described in the community policy, I can add a link in the LTS word here, would that be ok?

Reply:
Link added